WordPress/src/main/java/org/wordpress/android/sharedlogin/resolver/SharedLoginResolver.kt

```diff
@@ -50,6 +53,13 @@ class SharedLoginResolver @Inject constructor(
             val accessToken = queryResult.getValue(accessTokenCursor) ?: ""
             if (accessToken.isNotEmpty()) {
                 sharedLoginAnalyticsTracker.trackLoginSuccess()
+
+                if (jetpackAppMigrationFlowUtils.isFlagEnabled()) {
+                    dispatchUpdateAccessToken(accessToken)
+                    jetpackAppMigrationFlowUtils.startJetpackMigrationFlow()
```

Comment:
Curious if the intention is to leave this here temporarily (for convenience while developing)? Since we dispatch the token further on in the chained flow, we probably only want to do this once.

Fyi, this entire class is likely to be [refactored into an orchestrator](https://github.com/wordpress-mobile/WordPress-Android/pull/17399#pullrequestreview-1164158221), so we can tackle the kick-off of the UI screen flow from there - though it may happen _prior_ to dispatching the token.

Reply:
This was indeed only meant to be temporary, hence the fact that it actually "breaks" the continuation of data sync flow we have at the moment (user flags, reader saved posts).

The feature flag is also most probably just a temporary solution to enable testing the PRs for the UI.